source/kraken/tests/fill_disruption_from_chaos_tests.cpp

```diff
@@ -386,3 +386,40 @@ BOOST_AUTO_TEST_CASE(add_impact_on_line_over_midnigt_2) {
     auto* vj = b.data->pt_data->vehicle_journeys_map["vj:1"];
     BOOST_CHECK_MESSAGE(ba::ends_with(vj->adapted_validity_pattern->days.to_string(), "010110"), dump_vj(*vj));
 }
+
+BOOST_AUTO_TEST_CASE(add_impact_multi_stop_points_on_stop_area) {
```

Comment:
even if the test is quite easy to read, you need to add a block comment here, to explain a bit the test